.github/workflows/cla.yml

```diff
@@ -18,14 +18,11 @@ jobs:
           PERSONAL_ACCESS_TOKEN : ${{ secrets.CLA_ACCESS_TOKEN }}
         with:
           path-to-signatures: 'CLA/signatures/v1.0/cla.json'
-          path-to-document: 'https://github.com/corgibytes/freshli-cli/blob/main/CLA/CLA-v1.0.md'
+          path-to-document: 'https://github.com/corgibytes/freshli/blob/main/CLA/CLA-v1.0.md'
           branch: 'main'
           allowlist: dependabot[bot]
-
-          #Configurable inputs
-          #remote-organization-name: enter the remote organization name where the signatures should be stored (Default is storing the signatures in the same repository)
-          #remote-repository-name:  enter the  remote repository name where the signatures should be stored (Default is storing the signatures in the same repository)
+          remote-organization-name: 'corgibytes'
+          remote-repository-name: 'freshli-lib-cla'
```

Comment:
Instead of `freshli-lib-cla`, I wonder if it makes more sense to use `freshli-cla` as the repository where the CLA document and the signatures are housed. We could simply rename the `freshli-lib-cla` repository in that case.

Reply:
That would look cleaner :+1: I'm renaming it